Resources/Prototypes/XenoArch/Effects/normal_effects.yml

```diff
@@ -120,12 +120,12 @@
   targetDepth: 0
   effectHint: artifact-effect-hint-creation
   components:
-  - type: SpawnArtifact
-    maxSpawns: 20
-    spawns:
-    - id: FoodBanana
-      amount: 2 # Frontier 3<2
-      maxAmount: 4 # Frontier 6<4
+  # - type: SpawnArtifact
+    # maxSpawns: 20
+    # spawns:
+    # - id: FoodBanana
+      # amount: 3
+      # maxAmount: 6
```

Comment:
Why?

Reply:
The stupid infinite banana bug, also food.

Reply:
You don't want free food, so you add a whole new effect that... spawns food?! Mind explaining that a bit further?

Reply:
As said under, will edit.